### PR TITLE
Changed the warning for log retention to a full separate sub-header

### PR DESCRIPTION
**Pull request by bcrisologo**
# What changed, and why it matters

Summary:
- Added the "Log Retention Policy" header to the `access-service-logs.rst` file
- Updated the wording to present a clearer boundary for the customer to be aware of (where [slack thread](https://aiven-io.slack.com/archives/C02DUK2BK1C/p1660232189801349)) says up to 14 days for certain services

### CODE REVIEW

File: docs/platform/howto/access-service-logs.rst [hunk 1]
```diff
@@ -15,8 +15,8 @@ There are three built-in ways to inspect service logs at Aiven:
 
 * `Aiven API <https://api.aiven.io/doc/#operation/ProjectGetServiceLogs>`_ endpoint is available for fetching the same information two above methods output, in case programmatic access is needed.
 
-.. Warning::
-
-    Service logs are stored only for a few days. Unless you are using :doc:`logs integration </docs/integrations>` to another service, older logs are not accessible.
+Log Retention Policy
+--------------------
+Service logs are generally retained for a few days, and in some cases up to 14 days maximum.  Unless you are using :doc:`logs integration </docs/integrations>` to another service, older logs are not accessible.
```
Comment (lborda) [inline docs/platform/howto/access-service-logs.rst:20]: @bcrisologo I would mention Aiven Services instead "some cases"

Service logs are generally retained for a few days, and for some Aiven Services up to 14 days maximum.  Unless you are using :doc:`logs integration </docs/integrations>` to another service, older logs are not accessible.

Comment (bcrisologo) [reply docs/platform/howto/access-service-logs.rst]: Updated pull request to apply the change above

**Review by ftisiot — CHANGES_REQUESTED** [review]
Thanks for this @bcrisologo,
Could you please check the warning on the build? The title should be in sentence case

**Comment by bcrisologo** [comment]
Thanks @ftisiot! I've updated the title to sentence case.

**Review by ftisiot — APPROVED** [review]
LGTM, thanks!

**Pull request merged**